src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/RemoveBorder/RemoveBorderEffect.uwp.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Xamarin.CommunityToolkit.Effects;
+using Xamarin.Forms.Platform.UWP;
+using Effects = Xamarin.CommunityToolkit.Android.Effects;
+
+[assembly: Xamarin.Forms.ExportEffect(typeof(Effects.RemoveBorderEffect), nameof(RemoveBorderEffect))]
+
+namespace Xamarin.CommunityToolkit.Android.Effects
+{
+	public class RemoveBorderEffect : PlatformEffect
+	{
+		Thickness oldBorderThickness;
+
+		// Manipulate border thickness of UWP control
+		// https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control
```

Comment:
can you remove those lines? Or just move the link to the be above the class declaration

Reply:
Will just remove them.